.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,84 @@
+name: "Bug report 🐛"
+description: "Create a report to help us fix something that is currently broken."
+title: "[BUG] - <title>"
+labels: ["type: bug 🐛"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Welcome 👋
+
+        Thanks for using conda-store and taking some time to contribute to this project.
+
+        Please fill out each section below. This info allows maintainers to diagnose (and fix!) your issue as
+        quickly as possible.
+        Before submitting a bug, please make sure the issue hasn't been already addressed by searching through
+        the past issues in this repository.
+
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the bug is.
+        We suggest using bullets (indicated by * or -).
+      placeholder: Be as precise as you can.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: |
+        A clear and concise description of what you expected to happen.
+        We suggest using bullets (indicated by * or -).
+    validations:
+      required: true
```

Comment:
Is it really necessary to make all these fields required? I tend to find these sorts of bug report templates kind of redundant. A competent issue reporter will include this information when they "describe the bug". When the redundant fields are also required, it's really annoying.